server/work-queue/jobs/send-email.js

```diff
@@ -0,0 +1,48 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+var _ = require('underscore');
+var sendmail = require('sendmail')();
+
+var log = require('../../logger')('server.work-queue.jobs.send-email');
+var config = require('../../config');
+
+var contentTmpl = _.template('Welcome to Chronicle, <%= email %>!');
+
+module.exports = {
+  // o is an object with keys { email }
+  perform: function(o, cb) {
+    var toEmail = o.email;
+    log.verbose('job created with params ' + JSON.stringify(o));
+
+    switch (config.get('env')) {
+      case 'local':
+        sendmail({
+          from: 'info@chronicle.firefox.com',
```

Comment:
I'm pretty sure we will need to vary the 'from' address. Maybe that should be a sendmail config?